query.js

```diff
@@ -519,21 +522,45 @@ exports.search = (req, res, next) => {
 
 }
 
+/*
+ *  Given an entry's travels array and total number of tours, this function
+ *  returns the sum of all tour times.
+ */
+function getTravelTime(travels, numTours) {
+    
+    var accum = 0;
+    var i;
+    for (i = 1; i <= numTours; i++) {
+        var thisTour = travels.filter(function (d) {
+            return d.tourIndex == i;
+        });
+
+        if (thisTour && thisTour[0]) {
+            var numTravels = thisTour.length;
+            var firstTravel = thisTour[0];
+            var lastTravel = thisTour[numTravels - 1];
+
+            if (firstTravel.tourStartFrom && firstTravel.travelStartMonth && lastTravel.tourEndTo && lastTravel.travelEndMonth) {
+                var firstDate = new Date(firstTravel.tourStartFrom, firstTravel.travelStartMonth);
+                var lastDate = new Date(lastTravel.tourEndTo, lastTravel.travelEndMonth);
+    
+                accum += lastDate - firstDate;
```

Comment:
This was changed to tourStartFrom and tourEndTo because of a data error that said Edward Bird (422)'s last travel was in 1794 but last tour date was 1734, making his dot noticeably larger

Reply:
Got it, thanks